bakerydemo/templates/base/include/header-hero.html

```diff
@@ -3,16 +3,8 @@
 {% if page.image %}
     {% image page.image fill-1920x600 as image %}
     <div class="container-fluid hero" style="background-image:url('{{ image.url }}')">
-    <div class="hero-gradient-mask"></div>
-        <div class="container">
-            <div class="row">
-                <div class="col-md-7">
-                    <h1>{{ page.title }}</h1>
-                    {% if page.subtitle %}
-                        <p class="stand-first">{{ page.subtitle }}</p>
-                    {% endif %}
```

Comment:
Should we not retain the subtitle? Is it displayed elsewhere ?

Reply:
I've double checked all the references to this component, and the subtitle is not part of any of their page models. These pages typically render an `introduction` field instead, which is included elsewhere in their page template.

Reply:
This is because you switched `blog_page`’s header template from `header-hero.html` to `header-blog.html`. So we’d still need to make sure `header-blog.html` renders the subtitle.